Agents/MOPs_Assembler/mop_operations.py

```diff
@@ -0,0 +1,114 @@
+__author__ = "Aleksandar Kondinski"
+__license__ = "MIT"
+__version__ = '1.0.0'
+__status__ = "production"
+
+import csv
+import os
+import json
+from datetime import datetime
+from mop_am_rescaler import AssemblyModelRescaler
+from mop_am_gbu_positions import PositionExtractor
+from mop_cbu_aligner import GenericCBUProcessor
+from mop_jsons2xyz import JSONToXYZConverter
+
+class AssemblerWorkflow:
+    def __init__(self, assembly_model_file, cbus_dir, output_dir):
+        self.assembly_model_file = assembly_model_file
+        self.cbus_dir = cbus_dir
+        self.output_dir = output_dir
+
+    def read_csv(self, file_path):
```

Comment:
Please rename this function. It does not just read a CSV file - it runs the whole process! So, you may want to consider a `run` function, potentially as well as a separate `read` function.